src/webportal/src/app/job/job-view/fabric/job-detail/components/task-role-container-list.jsx

```diff
@@ -328,7 +328,9 @@ export default class TaskRoleContainerList extends React.Component {
                       name: 'Full log',
                       iconProps: {iconName: 'TextDocument'},
                       disabled: isNil(item.containerId),
-                      onClick: () => this.showContainerLog(`${item.containerLog}user.pai.all`, 'User logs (Last 4096 bytes)'),
+                      onClick: () => this.showContainerLog(
+                        `${item.containerLog}user.pai.all`,
+                        'User logs (Last 4096 bytes. Notice: The logs may out of order when merging stdout & stderr streams)'),
```

Comment:
out of time order?

Reply:
I'm not sure if `out of time order` will describe this issue correctly. It's actually out of code order.
And I think `out of order` already describe this issue cleanly.